app/(backend)/factories/serviceFactory.ts

```
@@ -0,0 +1,83 @@
+ * handling their dependency injection needs.
+ */
+export class ServiceFactory {
+  private static providerService: ProviderService | null = null;
+  private static apiKeyService: IApiKeyService | null = null;
+  private static modelService: IModelService | null = null;
+  
+  /**
+   * Get ApiKeyService instance (singleton)
+   */
+  static getApiKeyService(): IApiKeyService {
+    if (!this.apiKeyService) {
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)